test/regression/targets/sc-48428.cc

```diff
@@ -0,0 +1,86 @@
+#include <climits>
+
+#include <tiledb/tiledb>
```

Comment:
Let's add a comment somewhere that explains that this adds a var sized attribute encoded with the GZIP filter and that we write data that will result in a 0 sized var tile.

Reply:
> that will result in a 0 sized var tile

On second look this is quite non-apparent. How about I remove this regression test and write a targeted unit test for compressing a zero-sized buffer with the GZip filter?

Reply:
Updated the test's title, hope that's sufficient.